### ISSUE TITLE
Using c3-angular with require.js - problem!!

### ISSUE THREAD

**Issue by marco-faustinelli** [jettro/c3-angular-directive#83]
Hello, I am introducing require.js modularisation in my app, which uses c3-angular.

I get an error `c3 is not defined` which puzzles me a lot, because I have specified that c3-angular depends on c3.

Here are a couple of my requirejs shims for c3-angular. Have you got any experience with this sort issues?

    'c3-angular' : { deps: ['angular','c3'] },
    'c3' : { deps: ['d3'], exports: 'c3' },   

One of the major reasons of trouble in this case is that all c3-angular code is minified.
An uncompressed version would help me enormously with analyzing this problem, and I am opening another issue just for that.

**Comment by jettro** [comment]
Hm, sorry, I have o experience with requirejs. If I can find the time I'll try to investigate. At the moment I ignore the non minified version. I do not really see a problem to include it. Will do that in the next release. The non minified version is in the examples folder by the way. So you can check it. Also in browsers the .map file is used to make the minified version readable again.

**Comment by pascalbaljet** [comment]
+1

**Comment by jettro** [comment]
The latest release does contain the non minified version. Since I have no experience with RequireJS I am not sure a fix will come soon from my hand. Any help would be nice
